public/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
+Disallow: /settings
+Disallow: /#/*/settings
+
+Sitemap: https://snapshot.org/sitemap.xml
```

Comment:
/settings and https://snapshot.org/sitemap.xml pages doesn't exists actually, any reason to add these?

Reply:
@bonustrack pure distraction thanks! I assumed the existence of a sitemap, I will correct it. Thanks again